### DISCUSSION TITLE
Application groups are not updated with files

### DISCUSSION THREAD

**Discussion by dashke** [tnodir/fort#111] [Q&A] [answered]
When you add files to the Application groups from the Programs window, the Application groups remain empty.

**Comment by tnodir** [comment, marked as answer]
I think, it's a misunderstanding of App Groups and Programs here.

Let me explain how they work per some app (e.g. `C:\test.exe`):

- you can add the app to _several_ App Groups and to Programs independently from each other
- higher App Group's settings will override the lower ones for the same app
- Programs settings will override the App Group's for the same app
- app's `group` setting in the Programs is independent from other App Groups for the same app
